graylog2-web-interface/src/views/logic/fieldactions/ChangeFieldType/IndexSetsTable.tsx

```diff
@@ -110,56 +108,43 @@ const IndexSetsTable = ({ field, setIndexSetSelection, fieldTypes, initialSelect
     setIndexSetSelection(newSelection);
   }, [setIndexSetSelection]);
 
-  const toggleDetailsOpen = useCallback(() => {
-    setShowDetails((cur) => !cur);
-  }, []);
-
   if (isLoadingLayoutPreferences || isLoading) {
     return <Spinner />;
   }
 
   return (
-    <>
-      <Button label={showDetails ? 'Hide index sets' : 'Show index sets'} bsStyle="link" className="btn-text" bsSize="xsmall" onClick={toggleDetailsOpen}>
-        <Icon name={`caret-${showDetails ? 'down' : 'right'}`} />&nbsp;
-        {showDetails ? 'Hide index sets' : 'Show index sets'}
-      </Button>
-      {
-          showDetails && (
-            <Container>
-              <PaginatedList onChange={onPageChange}
-                             totalItems={pagination?.total}
-                             pageSize={layoutConfig.pageSize}
-                             activePage={activePage}
-                             showPageSizeSelect={false}
-                             useQueryParameter={false}>
-                {!list?.length && (
-                  <NoEntitiesExist>
-                    No index sets have been found.
-                  </NoEntitiesExist>
-                )}
-                {list.length && (
-                  <EntityDataTable<FieldTypeUsage> activeSort={layoutConfig.sort}
-                                                   bulkSelection={{
-                                                     onChangeSelection,
-                                                     initialSelection,
-                                                     actions: renderBulkActions,
-                                                   }}
-                                                   columnDefinitions={attributes}
-                                                   columnRenderers={columnRenderers}
-                                                   columnsOrder={DEFAULT_LAYOUT.columnsOrder}
-                                                   data={list}
-                                                   onColumnsChange={onColumnsChange}
-                                                   onPageSizeChange={onPageSizeChange}
-                                                   onSortChange={onSortChange}
-                                                   pageSize={layoutConfig.pageSize}
-                                                   visibleColumns={layoutConfig.displayedAttributes} />
-                )}
-              </PaginatedList>
-            </Container>
-          )
-        }
-    </>
+    <Container>
+      <ControlLabel>Select targeted index sets</ControlLabel>
```

Comment:
Sometimes the titles in this modal are capitalized, sometimes they are not. It would be nice to unify their styling by capitalizing all headlines.